core/tests/puppeteer-acceptance-tests/utilities/user/exploration-editor.ts

```diff
@@ -290,6 +297,18 @@ export class ExplorationEditor extends BaseUser {
     }
   }
 
+  /**
+   * This function Waits for the autosave indicator to appear and then disappear.
+   */
+  async waitForAutosaveIndicator(): Promise<void> {
+    await this.page.waitForSelector('span.e2e-test-autosave-indicator', {
```

Comment:
Please store it as a constant at the beginning of this file. Ditto for the one below.

Reply:
Done